app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/finance/MoneyUtils.scala

```diff
@@ -115,11 +115,12 @@ object MoneyUtils {
   def amountInPoundsYesNoForm(
     optionId: String,
     valueId: String,
-    maybeInnerOption: Option[InnerOption[BigDecimal]] = None
+    maybeInnerOption: Option[InnerOption[BigDecimal]] = None,
+    requiredErrorArgs: Seq[String] = Seq.empty
   ): Form[BigDecimal] = {
     val innerOption = maybeInnerOption.getOrElse(InnerOption { data =>
       FormUtils
-        .readValue(valueId, data, identity)
+        .readValue(valueId, data, identity, requiredErrorArgs)
```

Comment:
if you don't need this you may want to take it out - the required error in this `innerOption` and the required error in the `ConditionalRadioUtils.formatter` are for different things and we may want to pass in arguments to those things independently of each other. The `innerOption` looks for errors on the inner field when you select `Yes` on the form - `ConditionalRadioUtils.formatter` looks for errors on the yes/no part of the form